utils/api/configurations.js

```diff
@@ -63,7 +63,12 @@ export const configureRequests = ({ data, path }) => {
 // takes an array of errors for each page or component
 export const configureErrors = (errors) => {
   const env = process.env.NODE_ENV
-  const remainingErrors = errors.filter(error => error)
+  const remainingErrors = errors
+    .filter(error => error)
+    .map(error => ({
+      ...error,
```

Comment:
I'm unsure why we weren't getting the `response` property before, but something about this spread operator version of `error` returns it. 🤷🏾‍♀️